shared/js/background/startup.js

```diff
@@ -56,6 +56,12 @@ export async function onStartup () {
 
             // now setup extension listeners
             newTabTrackerStats.register()
+
+            // install debugging tools in devMode
+            const devMode = (await browserWrapper.getFromSessionStorage('dev')) || false
+            if (devMode) {
+                require('./newtab-tracker-stats-debug.js').installDebug(newTabTrackerStats, trackerStats)
+            }
```

Comment:
I think you can just put this in `devbuild.js`, then it will also not be included in production builds. This also prevents this from failing if one somehow sets the `dev` flag on a production build where `self.dbg` is not set.

Reply:
Moved :)